src/patternfly/components/Table/table-scrollable.scss

```
@@ -1,46 +1,57 @@
 .pf-c-table {
-  --pf-c-table__sticky-column--cell-min-width--base: #{pf-size-prem(200px)};
-  --pf-c-table__sticky-column--MinWidth: var(--pf-c-table__sticky-column--cell-min-width--base);
-  --pf-c-table__sticky-column--Left: 0;
-  --pf-c-table__sticky-column--ZIndex: var(--pf-global--ZIndex--md);
-  --pf-c-table__sticky-column--Top: 0;
-  --pf-c-table__sticky-column--BackgroundColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-table__sticky-column--m-border-right--before--BorderRightWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-table__sticky-column--m-border-right--before--BorderRightColor: var(--pf-global--BorderColor--100);
-  --pf-c-table__sticky-column--m-border-left--before--BorderLeftWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-table__sticky-column--m-border-left--before--BorderLeftColor: var(--pf-global--BorderColor--100);
-  --pf-c-table--m-sticky-header--thead__sticky-column--ZIndex: calc(var(--pf-c-table__sticky-column--ZIndex) + 1);
-
-  .pf-c-table__sticky-column {
+  --pf-c-table__sticky--cell-min-width--base: #{pf-size-prem(200px)};
```

Looking at our other components, if the "base" var is modifying a css property name (versus a concept, like "spacer"), it would be written as

```suggestion
  --pf-c-table__sticky--cell--MinWidth--base: #{pf-size-prem(200px)};
```